src/pages/home/sub-components/populer-products.js

```diff
@@ -1,15 +1,16 @@
-import { productList } from "./data"
 import { ProductWrapper, ProductBody, ProductImage, ProductActions, Button, IconButton } from "react-felix-ui"
 import { AiOutlineHeart, MdAdd } from "@icons"
-
+import { useProducts } from "../../../store/providers/product-provider"
 const PopularProducts = () => {
+    const productsList = useProducts()
+    console.log(productsList)
     return (
-        productList.slice(0, 10).map((item, i) => {
+        productsList.slice(0, 10).map((item, i) => {
```

Comment:
Instead of using `item.name`, `item.img` , etc, here you can de-structure `item` & use it.